src/vmm/src/devices/virtio/net/test_utils.rs

```diff
@@ -395,7 +395,7 @@ pub mod test {
         pub fn get_default() -> TestHelper<'a> {
             let mut event_manager = EventManager::new().unwrap();
             let mut net = default_net();
-            let mem = single_region_mem(MAX_BUFFER_SIZE);
+            let mem = single_region_mem(2 * MAX_BUFFER_SIZE);
```

Comment:
Just out of curiosity, why do we need to double it?

Reply:
because in the `test_tx_big_frame` unit test we are trying to add a descriptor in that is bigger than `MAX_BUFFER_SIZE`. The queue logic will catch it if the guest tried to pass us a descriptor that points outside of the allocated guest memory, so we won't hit the control we're checking against here.